src/components/signup/signup.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "bootstrap/dist/css/bootstrap.css";
```

Comment:
Same comment as above. Avoid importing bootstrap everywhere. Once you import it above in the index.css that is added to your index.js which is the entry point of the application, then other parts of the application pick it up.